models/UNet2dExample.model.yaml

```
@@ -14,17 +14,17 @@ authors:
 cite:
     - text: "Ronneberger, Olaf et al. U-net: Convolutional networks for biomedical image segmentation. MICCAI 2015."
       doi: https://doi.org/10.1007/978-3-319-24574-4_28
+    - text: "2018 Data Science Bowl"
+      doi: https://www.kaggle.com/c/data-science-bowl-2018
 
 git_repo: https://github.com/bioimage-io/configuration/tree/master/models
-tags: [unet2d, pytorch, nucleus-segmentation]
+tags: [unet2d, pytorch, nucleus-segmentation, data-science-bowl-2018]
```

For the display, shorter tags are preferred, how about the following:
```suggestion
tags: [unet2d, pytorch, nucleus, segmentation, dsb2018]
```